dbms/src/Functions/FunctionsGrouping.h

```diff
@@ -146,31 +158,58 @@ class FunctionGrouping : public IFunctionBase
 
     ResultType groupingImplModeAndBit(UInt64 grouping_id) const
     {
-        return (grouping_id & meta_grouping_id) != 0;
+        UInt64 res = 0;
+        for (auto one_grouping_id : meta_grouping_ids)
+        {
+            res <<= 1;
+            if ((grouping_id & one_grouping_id) <= 0)
```

Comment:
Why `(grouping_id & one_grouping_id) <= 0`, I think `(grouping_id & one_grouping_id) < 0` is always false, so the condition could be `(grouping_id & one_grouping_id) == 0`?

Reply:
yes, I make `!((grouping_id & one_grouping_id) > 0)`, this is the lint change